.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-    "stage": 3,
+    "stage": 2,
```

Comment:
This was discussed. The object spread operator, `...`, is going to factor into JSX components from React beyond 0.14.* so I'm okay with this.